xikolo-ios/ViewControllers/Courses/CourseDetailViewController.swift

```diff
@@ -18,21 +19,41 @@ class CourseDetailViewController: UIViewController {
     @IBOutlet weak var dateView: UILabel!
     @IBOutlet weak var teacherView: UILabel!
     @IBOutlet weak var descriptionView: UITextView!
-
+    @IBOutlet weak var enrollmentButton: SimpleRoundedButton!
+    
+    @IBAction func enrollmentButtonClick(_ sender: SimpleRoundedButton) {
```

Comment:
Since this is an IBAction and the sender is a button, I'd personally prefer just the action as a verb for the function name